applications/peripheral_sensor_node/configuration/nrf54l15pdk_nrf54l15_cpuapp/app.overlay

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
All other are 2023. Please align either all to 2024 or this to 2023

Reply:
New files are 2024. those created in the next repository are 2023.